crates/re_space_view_bar_chart/src/view_part_system.rs

```diff
@@ -25,6 +25,19 @@ impl IdentifiedViewSystem for BarChartViewPartSystem {
     }
 }
 
+struct BarChartVisualizerEntityFilter;
+
+impl VisualizerAdditionalApplicabilityFilter for BarChartVisualizerEntityFilter {
+    fn update_applicability(&mut self, event: &re_arrow_store::StoreEvent) -> bool {
+        event.diff.cells.iter().any(|(component_name, cell)| {
+            component_name == &re_types::components::TensorData::name()
+                && re_types::components::TensorData::from_arrow(cell.as_arrow_ref())
+                    .map(|tensors| tensors.iter().any(|tensor| tensor.is_vector()))
+                    .unwrap_or(false)
```

Comment:
this seems likely to be a recurring pattern we'll want a helper for at some point.

Reply:
adding some form of utility that's generic over the component type!